l3kernel/l3int.dtx

```diff
@@ -2050,7 +2051,7 @@
   {
     \int_compare:nNnTF {#1} > {#2}
       {
-        \exp_args:NNo \exp_args:No \@@_to_symbols:nnnn
```

Comment:
This change changes behaviour if `#3` contains material that might expand further (which I didn't check).

Reply:
I think `#3` here is supposed to comprise character tokens: lets see what others think.